tests/_async/test_mapping.py

```diff
@@ -17,12 +17,11 @@
 
 import json
 
-from elasticsearch_dsl import Keyword, Nested, Text, analysis
-from elasticsearch_dsl._async import mapping
+from elasticsearch_dsl import AsyncMapping, Keyword, Nested, Text, analysis
```

Comment:
I could have done that, but if we are saying we prefer direct imports I thought it would be better to switch. I still think it doesn't hurt to have the async imports in `mapping`, `search`, etc. if anything for symmetry, since obviously we cannot remove the sync classes from there.

Reply:
No real reason, I guess I just felt bad about you doing more work than necessary. Resolving.